lms/static/styles/components/_Table.scss

```diff
@@ -10,6 +10,10 @@ $table-item-height: 40px;
   border: 1px solid var.$grey-6;
 
   margin-top: 10px;
+
+  &.has-files {
+    min-height: 176px;
```

Comment:
Where does this very specific value of 176px come from?

Reply:
OK, I think we should at least have a short comment here to explain what this rule is intended to do. Can you see if there a way of calculating this value from other variables or note how you determined it. Otherwise this could presumably get out of sync if other variables or properties are changed in future.